plsql/ExcelGen.pkb

```diff
@@ -1512,6 +1514,9 @@ create or replace package body ExcelGen is
       when dbms_sql.CLOB_TYPE then
         dbms_sql.define_column(p_cursor_number, i, data.clob_value);
         columnItem.supertype := ST_LOB;
+      when dbms_sql.BINARY_DOUBLE_TYPE then
+        dbms_sql.define_column(p_cursor_number, i, data.binary_double_value);
+        columnItem.supertype := ST_BINARY_DOUBLE;
```

Comment:
Should be ST_NUMBER

Reply:
Are you saying that when the input data type is dbms_sql.BINARY_DOUBLE_TYPE, that we can do a define_column with data.number_value and set columnItem.supertype to ST_NUMBER?  Let Oracle do the conversion from binary_double to number when it pulls the values from the resultset into the array?  I'm not certain how that is implemented internally and did not know it was an option.  I can try it.

Reply:
No, I'm just saying that we don't need a specific ST_BINARY_DOUBLE **supertype**.
The BINARY_DOUBLE data type still have to be handled with the proper define_column and column_value overloads, just like you did.
The supertype info is only there to mutualize XML generation patterns.

So, this should be fine : 
```
      when dbms_sql.BINARY_DOUBLE_TYPE then
        dbms_sql.define_column(p_cursor_number, i, data.binary_double_value);
        columnItem.supertype := ST_NUMBER;
```

then in createWorksheetImpl, no need to add a ST_BINARY_DOUBLE case.

Reply:
It turns out that oracle will do the conversion just fine into data.number_value when it copies from the internal dbms_sql result array. That makes it simpler. Any conversion errors would happen during the dbms_sql call rather than when we convert to string or number in ExcelGen package though.  I can put data.binary_double_value back into play if you prefer.